web/test/pages/profile/profile-starting.test.tsx

```diff
@@ -41,7 +42,11 @@ import {
   OperatingPhases,
   UserData,
 } from "@businessnjgovnavigator/shared/index";
-import { generateFormationData, generateTaxFilingData } from "@businessnjgovnavigator/shared/test";
+import {
+  generateFormationData,
```

Comment:
So I wonder if maybe these tests belong in profile-shared with an it each to test against each persona. As they all feel like they're repeating the same test.

Reply:
I can see a situation in the future where we want to change this slightly and have to modify it across all these files as opposed to 1 test in shared

Reply:
I agree that if there is a way to consolidate the repeated tests, then we should do it and place it in the shared file to decrease the amount of places you would have to go to update logic if the behavior were to change.